lms/djangoapps/ccx/views.py

```diff
@@ -349,6 +349,9 @@ def visit(node, depth=1):
         widgets, which use text inputs.
         """
         for child in node.get_children():
+            # in case the children are visible to staff only, skip them
```

Comment:
Why? You should also update the docstring of this function to elaborate that `visti` only visits student visible items